assignments/arrays.js

```diff
@@ -65,14 +65,15 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // The dealer can't recall the information for a car with an id of 33 on his lot. 
 //Help the dealer find out which car has an id of 33 by logging the car's year, make, and model 
 //in the console log provided to you below:
-console.log(`Car 33 is a 2011 Jeep` );
+console.log('Car 33 is a ' + inventory[32].car_year + ' '+ inventory[32].car_make +' '+ inventory[32].car_model + '.');
+
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
+let lastCar = inventory[inventory.length-1];
 for (let i=0; i<inventory.length; i++) {
-    lastCar
+    console.log(lastCar.car_make + ' ' + lastCar.car_model)
```

Comment:
You've already accessed your last car in line 72, so with your loop you're actually re-sending the same information in rapid succession! Revisit this part